lib/views/home.dart

```diff
@@ -1,10 +1,189 @@
+// ignore_for_file: prefer_const_constructors
```

Comment:
Do not use comments to override the errors . It is not good style of coding. Remove comments and resolve the issue